freeplane.json

```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://freeplane.sourceforge.net/wiki/index.php/Main_Page",
+    "version": "1.7.3",
+    "hash": "sha1:30e3893d1855adb4db44f8f3ff74150c6476035c",
+    "description": "Freeplane is a free and open source software application that supports thinking, sharing information and getting things done at work, in school and at home. The software can be used for mind mapping and analyzing the information contained in mind maps.",
+    "license": "GNU GPL V2+",
+    "url": "https://downloads.sourceforge.net/project/freeplane/freeplane%20stable/archive/1.7.3/freeplane_bin-1.7.3.zip",
+    "bin": "freeplane.exe",
+    "extract_dir": "freeplane-1.7.3",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "shortcuts": [
+        [
+            "freeplane.exe",
+            "Freeplane"
+        ]
+    ],
+    "checkver": {
```

Comment:
Could be changed to:

```json
    "checkver": {
        "url": "https://www.freeplane.org/info/history/history_en.txt",
        "re": "=+\\s([\\d.]+)\\s=+"
    },
```

Reply:
Changed regex a bit, I hope it's ok now.